src/main/java/exceptions/EmptyDescriptionException.java

```diff
@@ -0,0 +1,23 @@
+package exceptions;
+
+/**
+ * Throws EmptyDescriptionException when user does
+ * not specify a description after an action.
+ *
+ * @author ChesterSim
+ */
+public class EmptyDescriptionException extends Exception {
+    private String taskType;
+
+    public EmptyDescriptionException(String taskType) {
+        super();
+        this.taskType = taskType;
+    }
+
+    @Override
+    public String toString() {
+        String taskType;
```

Comment:
I think you accidentally included this unused piece of code. Can consider removing this!